vochain/app.go

```diff
@@ -75,10 +74,18 @@ type BaseApplication struct {
 	circuitConfigTag    string
 	dataDir             string
 	genesisInfo         *tmtypes.GenesisDoc
+
+	// testMockBlockStore is used for testing purposes only
+	testMockBlockStore *testutil.MockBlockStore
```

Comment:
seems strange to declare this here (just for testing purposes), but I am assuming it cannot be declared anywhere else.